python/tests/kernel/test_ir_operations.py

```diff
@@ -13,6 +13,14 @@
 import cudaq
 from cudaq import spin
 
+@pytest.fixture(autouse=True)
+def do_something():
+    if os.getenv("CUDAQ_PYTEST_EAGER_MODE") == 'OFF':
+        cudaq.enable_jit()
+    yield
+
+    if cudaq.jit_enabled(): cudaq.__clearKernelRegistries()
+    cudaq.disable_jit()
 
 def test_synthesize():
```

Comment:
Does the comment on line#26 still apply? (code on line#27 may not be needed)
Same with `jit=True` on line#37

Reply:
I'll restructure this one to only run in MLIR mode.